packages/react-server/src/ReactFizzHooks.js

```diff
@@ -616,8 +620,13 @@ function use<T>(usable: Usable<T>): T {
           }
         }
       }
-    } else {
-      // TODO: Add support for Context
+    } else if (
+      usable.$$typeof != null &&
```

Comment:
I didn't notice this in the previous PR, but this null check isn't necessary, right? If it's undefined or null, the check below will fail anyway.